### ISSUE TITLE
Remove "obsolete" files

### ISSUE THREAD

**Issue by dsenalik** [tripal/tripal#2046]
### Depends on #2036 (merged)
This is a reminder to delete some files that are no longer needed

1. `./tripal/src/Services/TripalPublish.php` - PR #1991 made this obsolete
2. `tripal/config/install/tripaldbx.settings.yml` - PR #2037
3. `tripal/config/schema/tripaldbx.schema.yml` - PR #2037

### Branch Name

tv4g0-issue2046-obsolete-file-removal
